apollo-router/src/plugins/fleet_detector.rs

```diff
@@ -0,0 +1,126 @@
+use schemars::JsonSchema;
+use serde::Deserialize;
+use sysinfo::System;
+use tokio::task::JoinHandle;
+use tower::BoxError;
+use tracing::debug;
+use tracing::info;
+
+use crate::plugin::Plugin;
+use crate::plugin::PluginInit;
+
+#[derive(Debug)]
+struct FleetDetector {
+    handle: JoinHandle<()>,
+}
+
+#[derive(Debug, Default, Deserialize, JsonSchema)]
+struct Conf {}
+
+#[async_trait::async_trait]
+impl Plugin for FleetDetector {
+    type Config = Conf;
+
+    async fn new(_: PluginInit<Self::Config>) -> Result<Self, BoxError> {
+        debug!("beginning environment detection");
+        debug!("spawning continuous detector task");
```

Comment:
Do we need these 2 debugs knowing that nothing happens between these 2 logs ? Maybe we can put everything into one log ?

Reply:
Quite right, that was a hangover from an older version, have shortened it to one :)